.circleci/config.yml

```diff
@@ -1,6 +1,7 @@
 version: 2.1
 orbs:
-  hmpps: ministryofjustice/hmpps@10
+  hmpps: ministryofjustice/hmpps@dev:alpha
```

Comment:
Will update once Andy's [PR](https://github.com/ministryofjustice/hmpps-circleci-orb/pull/196) is merged.

Reply:
Andy's PR has been merged so updated.